southbound/impl/src/main/java/org/opendaylight/ansible/southbound/AnsibleProcessListener.java

```diff
@@ -0,0 +1,35 @@
+package org.opendaylight.ansible.southbound;
+
+import ch.vorburger.exec.ManagedProcessListener;
+import org.opendaylight.yang.gen.v1.urn.ietf.params.xml.ns.yang.ietf.yang.types.rev130715.Uuid;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AnsibleProcessListener implements ManagedProcessListener {
+    private Uuid processUuid;
+    private AnsibleCommandServiceImpl commandService;
+    private static final Logger LOG = LoggerFactory.getLogger(AnsibleProcessListener.class);
+
+    public AnsibleProcessListener(AnsibleCommandServiceImpl commandService, Uuid processUuid) {
+        this.commandService = commandService;
+        this.processUuid = processUuid;
+    }
+
+    @Override
+    public void onProcessComplete(int i) {
+        try {
+            commandService.parseAnsibleResult(processUuid);
+        } catch (AnsibleCommandException e) {
+            LOG.error("Unable to complete process callback", e);
```

Comment:
what do we do for the apps looking for a response? Guess can't do much if the service call failed it would likely fail again.

Reply:
yeah would need to write FAILED but can probably ignore this for now